FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/ExternalAdRecommendationCell.swift

```diff
@@ -0,0 +1,310 @@
+import Foundation
+
+public class ExternalAdRecommendationCell: UICollectionViewCell, AdRecommendationCell, AdRecommendationConfigurable {
+    // MARK: - Internal properties
+
+    private static let titleHeight: CGFloat = 20.0
+    private static let titleHeightTwoLines: CGFloat = 40.0
+    private static let titleTopMargin: CGFloat = 3.0
+    private static let bottomMargin: CGFloat = 15.0
+    private static let subtitleHeight: CGFloat = 17.0
+    private static let ribbonTopMargin: CGFloat = 6.0
+    private static let ribbonHeight: CGFloat = 19.0
+    private static let subtitleTopMargin: CGFloat = 6.0
+    private static let accessoryHeight: CGFloat = 14.0
```

Comment:
These numbers seems very arbitrary. Are they specified in the design, or can we use our regular spacings for this?